src/ContractionTreeToGraph.jl

```diff
@@ -0,0 +1,120 @@
+
+"""The main object here is `g' a NamedGraph which represents a graphical version of a contraction sequence.
```

Comment:
I would put this into the doctstring of `create_contraction_tree`/`contraction_sequence_to_graph`.